source/fuzz/fuzzer.cpp

```diff
@@ -198,214 +215,233 @@ Fuzzer::FuzzerResultStatus Fuzzer::Run(
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceLoadsStoresWithCopyMemories>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceParameterWithGlobal>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceLinearAlgebraInstructions>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceIrrelevantIds>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceOpPhiIdsFromDeadPredecessors>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceOpSelectsWithConditionalBranches>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassReplaceParamsWithStruct>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassSplitBlocks>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    MaybeAddRepeatedPass<FuzzerPassSwapBranchConditionalOperands>(
+        &pass_instances, ir_context.get(), &transformation_context,
+        &fuzzer_context, transformation_sequence_out);
+    // There is a theoretical possibility that no pass instances were created
+    // until now; loop again if so.
+  } while (pass_instances.GetPasses().empty());
+
+  RepeatedPassRecommenderStandard pass_recommender(&pass_instances,
+                                                   &fuzzer_context);
+
+  std::unique_ptr<RepeatedPassManager> repeated_pass_manager = nullptr;
+  switch (repeated_pass_strategy_) {
+    case RepeatedPassStrategy::kSimple:
+      repeated_pass_manager = MakeUnique<RepeatedPassManagerSimple>(
+          &fuzzer_context, &pass_instances);
+      break;
+    case RepeatedPassStrategy::kLoopedWithRecommendations:
+      repeated_pass_manager =
+          MakeUnique<RepeatedPassManagerLoopedWithRecommendations>(
+              &fuzzer_context, &pass_instances, &pass_recommender);
+      break;
+    case RepeatedPassStrategy::kRandomWithRecommendations:
+      repeated_pass_manager =
+          MakeUnique<RepeatedPassManagerRandomWithRecommendations>(
+              &fuzzer_context, &pass_instances, &pass_recommender);
+      break;
+    default:
+      assert(false && "Invalid repeated pass manager.");
+      break;
```

Comment:
It seems you are handling all values from `RepeatedPassStrategy`. We could remove `default` block in this case. The advantage is that it will produce a compile-time error if some enumerator is missing from the switch as opposed to the runtime assertion failure.

Reply:
Done.